index.js

```diff
@@ -8,18 +8,18 @@ function main(stream) {
 	var render = function () {
 		cliCursor.hide();
 		var out = [].join.call(arguments, ' ') + '\n';
-		stream.write(ansiEscapes.eraseLines(prevLineCount) + out);
+		render.clear();
+		stream.write(out);
```

Comment:
Not sure if it matters, but seeing as I often use this in a hot loop and some terminals can be slow, I prefer to keep it to one stdout write, as previously.